### PR TITLE
Add warning before proceeding to crypto payment

### LINKED ISSUE #1911: Add some information regarding the crypto time lock
We should add some information to tell our users that the cryto price will be locked for 60min

### PR DESCRIPTION
**Pull request by Tbaut**
closes #1911

---
Submission checklist: 

   

#### Layout
- [x] Change looks good in the desktop web ui
- [x] Change looks good in the mobile web ui

#### Theme
- [x] Components / elements inspected in light mode 
- [x] Components / elements inspected in dark mode

### CODE REVIEW

File: packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/Common/ConfirmPlan.tsx [hunk 1]
```diff
@@ -291,6 +302,19 @@ const ConfirmPlan = ({
...
       </div>
+      {paymentMethod === "crypto" && <div className={classes.rowBox}>
+        <Typography
+          variant="body1"
+          component="p"
+          className={classes.warningText}
+        >
+          <InfoCircleIcon className={classes.icon} />
+          <Trans>
+            Once you proceed, your account is expected to make a payment within 60min. If no payment is received, your plan will not change.
```
Comment (tanmoyAtb) [inline packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/Common/ConfirmPlan.tsx]: is `60 minutes` better ?

**Review by tanmoyAtb — APPROVED** [review]
design looks good.

**Review by asnaith — APPROVED** [review]
Looks good. I agree that the copy change per Tanmoy's comment would be better

File: packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/Common/ConfirmPlan.tsx (hunk 1 above)
Comment (Tbaut) [reply packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/Common/ConfirmPlan.tsx]:
```suggestion
            Once you proceed, your account is expected to make a payment within 60 minutes. If no payment is received, your plan will not change.
```

File: packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/Common/ConfirmPlan.tsx [hunk 2]
```diff
@@ -291,6 +302,19 @@ const ConfirmPlan = ({
...
       </div>
+      {paymentMethod === "crypto" && <div className={classes.rowBox}>
+        <Typography
+          variant="body1"
+          component="p"
+          className={classes.warningText}
+        >
+          <InfoCircleIcon className={classes.icon} />
+          <Trans>
+            Once you proceed, your account is expected to make a payment within 60 minutes. If no payment is received, your plan will not change.
```
Comment (Tbaut) [inline packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/Common/ConfirmPlan.tsx]:
```suggestion
            Once you proceed, your account is expected to make a payment within 60 minutes. If no payment is received
            , your plan will not change.
```

**Pull request merged**